app/models/user.rb

```diff
@@ -42,8 +42,19 @@ def user_detail
     nil
   end
 
+  def user_submissions
+    users =
+      if gravity_user_id
+        User.where(gravity_user_id: gravity_user_id)
+      else
+        User.where(name: name, email: email, phone: phone)
```

Comment:
@iskounen Do we want to try to identify users by using such fields combination or just ignore such submissions?

Reply:
@iskounen @kiryl-zubarau I don't think this is very important, since I use this method only to display the number of submissions of a given user on the page in the Convection admin
<img width="1660" alt="Screen Shot 2021-11-23 at 6 57 11 PM" src="https://user-images.githubusercontent.com/21379857/143058700-b29144b1-93ca-40af-a329-15a2f70d6e7e.png">

Reply:
I think it's OK to display the number of submissions made by a user like this. This number will not always be accurate for anonymous submissions, but I think that it is still valuable for admins to see an approximation of how many submissions have been made by someone with the same name + email + phone.